NYCJobsWeb/Controllers/HomeController.cs

```diff
@@ -72,10 +72,13 @@ public ActionResult Suggest(string term, bool fuzzy = true)
                 suggestions.Add(result.Text);
             }
 
+            // Get unique items
+            List<string> uniqueItems = suggestions.Distinct().ToList();
```

Comment:
I did this because I want to have distinct terms in the drop down (not due to SDK update).  I will change this when we have autosuggest in a few weeks.